lib/timeline/Stack.js

```diff
@@ -50,13 +53,18 @@ exports.stack = function(items, margin, force) {
     if (item.stack && item.top === null) {
       // initialize top position
       item.top = margin.axis;
+      var shouldBail = false;
 
       do {
         // TODO: optimize checking for overlap. when there is a gap without items,
         //       you only need to check for items from the next item on, not from zero
         var collidingItem = null;
         for (var j = 0, jj = items.length; j < jj; j++) {
           var other = items[j];
+          shouldBail = shouldBailItemsRedrawFunction();
```

Comment:
Is there any possibility of `shouldBailItemsRedrawFunction` being `null`, `undefined` or some other silly value?